content/handbook/people-group/guidance-on-feedback.md

```
@@ -0,0 +1,233 @@
+![Feedback Image](/images/handbook/Training/Feedback_Image.png)
+
+1. ポジティブな意図を持つ
+   - 誰かを傷つける意図があり、助ける意図がない場合は、フィードバックを与えることを再考してください。
+   - 厳しいフィードバックは、人々が成功し卓越するのに役立ちません。
+1. 頻繁なイベントにし、リアルタイムに行う
+   - 配信のためにパフォーマンスレビューを待たない
+1. 絶対的な表現を使わない
+   - 例: "You always."（あなたはいつも）"You never show up to meetings on time."（あなたはミーティングに時間通りに現れたことがない）
+1. フィードバックはポジティブと建設的の両方であるべき
+   - ポジティブと建設的の比率を 4:1 にすることを推奨します
+1. [Situation-Behavior-Impact (S-B-I) モデル](#s-b-i-model)を使用する
```

markdownlint-cli2 (0.22.1)

[warning] Link fragments should be valid

(MD051, link-fragments)